Component/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ endif()
 
 set(SOFACOMPONENT_SOURCE_DIR "src/sofa/component")
 
-set(SOFACOMPONENT_DIRS ODESolver IO Playback SceneUtility Topology Visual LinearSolver Mass Diffusion Mapping SolidMechanics StateContainer Constraint) # AnimationLoop Mapping Collision Engine UI Controller) # Device
+set(SOFACOMPONENT_DIRS ODESolver IO Playback SceneUtility Topology Visual LinearSolver Mass Diffusion Mapping SolidMechanics StateContainer Constraint AnimationLoop) # Collision Engine UI Controller) # Device
```

Comment:
is the comment needed?

Reply:
on the right ? it is just a kind of roadmap; the goal is to have nothing at the end